cmd/crane/cmd/validate.go

```diff
@@ -45,7 +48,11 @@ func NewCmdValidate(options *[]crane.Option) *cobra.Command {
 					return fmt.Errorf("failed to read image %s: %v", flag, err)
 				}
 
-				if err := validate.Image(img); err != nil {
+				opt := []validate.Option{}
+				if fast {
+					opt = append(opt, validate.Fast)
+				}
+				if err := validate.Image(img, opt...); err != nil {
 					fmt.Printf("FAIL: %s: %v\n", flag, err)
```

Comment:
Will this result in a different exit code?

Reply:
Ah yeah I neglected to follow up on https://github.com/google/go-containerregistry/pull/988#discussion_r617088024

Pulled this out into https://github.com/google/go-containerregistry/pull/1014